### PR TITLE
vivid: new package

### PR DESCRIPTION
**Pull request by Freed-Wu**

### CODE REVIEW

**Comment by Freed-Wu** [comment]
`failure: mingw-w64-vivid: missing source checksums` means missing sha256sums, really? So `b2sums` cannot be used?

**Comment by Biswa96** [comment]
Need some changes.

1. Add `(mingw-w64)` in pkgdesc.
2. provides and conflicts are not required here.
3. Only sha256sums or sha512sums is enough.
4. Install the files in `${MINGW_PREFIX}` instead of `/usr`.

**Comment by Freed-Wu** [comment]
Fixed

**Comment by Freed-Wu** [comment]
Now it can work.

**Comment by MehdiChinoune** [comment]
> `failure: mingw-w64-vivid: missing source checksums` means missing sha256sums, really? So `b2sums` cannot be used?

It should be acceptable!
https://wiki.archlinux.org/title/PKGBUILD#Integrity
Unless something is wrong with msys2-pacman

File: mingw-w64-vivid/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+pkgver=0.8.0
+pkgrel=1
+pkgdesc='A themeable LS_COLORS generator with a rich filetype datebase (mingw-w64)'
+arch=(any)
+mingw_arch=(mingw32 mingw64 ucrt64 clang64)
+url=https://github.com/sharkdp/vivid
+makedepends=($MINGW_PACKAGE_PREFIX-rust)
+license=(MIT Apache)
+source=("$url/archive/v$pkgver.tar.gz")
+checksums=('e58e0936db25c81ff257775463f1d422d97c706aec2d0134e39b62151ded23cb')
```
Comment (MehdiChinoune) [inline mingw-w64-vivid/PKGBUILD:15]: for what?

File: mingw-w64-vivid/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+_realname=vivid
+
+pkgbase=mingw-w64-$_realname
+pkgname=$MINGW_PACKAGE_PREFIX-$_realname
+pkgver=0.8.0
+pkgrel=1
+pkgdesc='A themeable LS_COLORS generator with a rich filetype datebase (mingw-w64)'
+arch=(any)
+mingw_arch=(mingw32 mingw64 ucrt64 clang64)
+url=https://github.com/sharkdp/vivid
```
Comment (MehdiChinoune) [inline mingw-w64-vivid/PKGBUILD:11]:
```suggestion
url='https://github.com/sharkdp/vivid'
```

File: mingw-w64-vivid/PKGBUILD (hunk 1 above)
Comment (Freed-Wu) [reply mingw-w64-vivid/PKGBUILD:15]: Oh, is a typo, I want to write sha256sum, but paste another word. I'll fix it.

File: mingw-w64-vivid/PKGBUILD (hunk 2 above)
Comment (Freed-Wu) [reply mingw-w64-vivid/PKGBUILD:11]: Both of them work same. Although I'll add quotes, I still wanna know why the latter is better? Because of code style?

Comment (MehdiChinoune) [reply mingw-w64-vivid/PKGBUILD]: Yes. because of code style.

**Pull request merged**